azure_li_services/defaults.py

```diff
@@ -26,6 +26,10 @@ class Defaults(object):
 
     Provides class methods for default values
     """
+    @classmethod
+    def get_config_file_name(self):
+        return '/etc/azure_li_config.yaml'
```

Comment:
Huh? I thought the decided name was "/etc/suse_firstboot_config.yaml".

I don't care for _li_ in the name, as that excludes vli (very large instance), as well as perhaps using this on Azure VMs. So if you don't want to use suse_firstboot_config.yaml, pick something that's not specific to a particular hardware line.

Also, original suggestion in #2 was `<lun_mount_point>/suse_firstboot_config.yaml` (i.e. `/suse_firstboot_config.yaml`, not in `/etc`). Again, I'm not picky, let's just be very explicit on what we want.

Reply:
yep now that we are clear on the name, I've updated the code